llm/llm.go

```diff
@@ -1,17 +1,20 @@
 package llm
 
-// #cgo CFLAGS: -Illama.cpp
-// #cgo darwin,arm64 LDFLAGS: ${SRCDIR}/build/darwin/arm64_static/libllama.a -lstdc++
-// #cgo darwin,amd64 LDFLAGS: ${SRCDIR}/build/darwin/x86_64_static/libllama.a -lstdc++
-// #cgo windows,amd64 LDFLAGS: ${SRCDIR}/build/windows/amd64_static/libllama.a -static -lstdc++
-// #cgo windows,arm64 LDFLAGS: ${SRCDIR}/build/windows/arm64_static/libllama.a -static -lstdc++
-// #cgo linux,amd64 LDFLAGS: ${SRCDIR}/build/linux/x86_64_static/libllama.a -lstdc++
-// #cgo linux,arm64 LDFLAGS: ${SRCDIR}/build/linux/arm64_static/libllama.a -lstdc++
+// #cgo CPPFLAGS: -Illama.cpp
+// #cgo LDFLAGS: -lstdc++ -lllama
+// #cgo darwin,arm64 LDFLAGS: -L${SRCDIR}/build/darwin/arm64_static/
+// #cgo darwin,amd64 LDFLAGS: -L${SRCDIR}/build/darwin/x86_64_static/
+// #cgo windows LDFLAGS: -static
+// #cgo windows,amd64 LDFLAGS: -L${SRCDIR}/build/windows/amd64_static/
+// #cgo windows,arm64 LDFLAGS: -L${SRCDIR}/build/windows/arm64_static/
```

Comment:
My suspicion is the windows build failure in CI is due to dropping -lstdc++ here.

Reply:
Actually, no, you have it a few lines up as a general setting, so it's not that... but the windows CI failure looks like it is related to your change.